### ISSUE TITLE
how to convert texture to vertices?

### ISSUE THREAD

**Issue by anewusername77** [abdallahdib/NextFace#47]
hi there, its me again.
by using `generateTextureFromAlbedo` in `morphablemodel.py` can convert color vertices to uv texures, how can i inverse this process to convert textures to color vertices?

thanks!

**Comment by abdallahdib** [comment]
U will have to use torch.nn.functional.grid_sample

x = morphableModel.uvMap[:, 0]
y = morphableModel.uvMap[:, 1]
map = torch.zeros_like(morphableModel.uvMap)
map[:, 0] = x
map[:, 1] = 1 - y #vertical axis is flipped
grid = (2 * map - 1) #must be betwee 0 and 1
torch.nn.functional.grid_sample(texture, grid)

I hope that helps

**Comment by anewusername77** [comment]
I'll try, learned a lot from you, thanks! 

With gratitude.
